### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_CreationAndDeployment_ScreenReader:During editing when Shift Plus TAB key is pressed a small dialog box pop ups on the page but Narrator does not announce anything.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#4202]
Environment Details:
Application Name: AzureNotebooks_Jupyter
Login URL: notebooks.azure.com
Browser Details:
Edge: Microsoft Edge 42.17134.1.0
Chrome: Version 70.0.3538.77 (Official Build) (64-bit)Operating System:Windows10

Steps to reproduce:
1.Login with the appropriate credentials to azure notebook account.    

2.Go to Get started à libraries à Select any of the libraries à click on clone to clone the library

3.Navigate to run button & hit enter --> Navigate to Creation and Deployment of an Azure ML Web Service & hit enter.
4. Navigate to Code Cell block.

Actual
During editing when Shift Plus TAB key is pressed a small dialog box pop ups on the page but Narrator does not announce anything for the end user to understand that various information is available for the end user to be aware of. 

Expected
During editing when Shift Plus TAB key is pressed a small dialog box pop ups on the page but Narrator should announce some information for the end user to understand that various information is available for the end user to be aware of. 

User Impact
End User will be unaware of the information present on the screen if that is not conveyed by the screen reader.

Recommendation
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

MAS Reference
https://microsoft.sharepoint.com/%3Aw%3A/r/teams/msenable/_layouts/15/WopiFrame2.aspx?sourcedoc=%7B54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4%7D

[A11y_AzureNotebooks_Jupyter_CreationAndDeployment_ScreenReader.zip](https://github.com/jupyter/notebook/files/2576089/A11y_AzureNotebooks_Jupyter_CreationAndDeployment_ScreenReader.zip)
[A11y_AzureNotebooks_Jupyter_CreationAndDeployment_ScreenReader_1.zip](https://github.com/jupyter/notebook/files/2576090/A11y_AzureNotebooks_Jupyter_CreationAndDeployment_ScreenReader_1.zip)

**Comment by jtpio** [comment]
Closing as there are now different issues tracking accessibility improvements for Jupyter Notebook 7, for example https://github.com/jupyter/notebook/issues/6800

Thanks!
